pkg/api/types_test.go

```diff
@@ -387,6 +387,49 @@ func TestMasterProfileIsUbuntuNonVHD(t *testing.T) {
 	}
 }
 
+func TestMasterProfileHasMultipleNodes(t *testing.T) {
```

Comment:
Here's how we would add functional unit test coverage, to the golang native func that implements the business logic